### ISSUE TITLE
Possible port to o3ds system?

### ISSUE THREAD

**Issue by kuwoh** [FWGS/xash3d#315]
im wondering if this was possible to port to the 3ds just like how quake was ported

**Comment by a1batross** [comment]
I don't know about 3ds so much and even never saw this console IRL. :)

**Comment by kuwoh** [comment]
3ds specs (old one not new)
CPU: 268 mhz dual core
Memory: RAM = 128 mb, VRAM = 6MB
Graphics:  DMP PICA200 GPU

**Comment by mittorn** [comment]
It seems to be possible. Just try to build and run it with XASH_NANOGL macro enabled if you have sdk and device. I think, there are working sdl2 3ds ports

**Comment by mittorn** [comment]
https://github.com/xerpi/SDL-3DS

**Comment by kuwoh** [comment]
i do not have the capability to do a port i was pretty much just asking if it's possible and if you guys could try on your spare time? i wouldn't mind being a tester

**Comment by mittorn** [comment]
I have not much spare time and will not interested as I do not have such device.
But port is possible in future.
Is there any emulator and sdk availiable?

**Comment by kuwoh** [comment]
yes there is an emulator available. it's called citra 3ds emulator
https://citra-emu.org/

**Comment by Midnoclose** [comment]
However, the citra emulator is still in develeopment. Some apps may work on citra and not an actual 3ds.

**Comment by kuwoh** [comment]
i don't mind using my 2ds as a tester

**Comment by Midnoclose** [comment]
I could use my n3ds as well.

**Comment by kuwoh** [comment]
Neat that they have two willing testers

**Comment by a1batross** [comment]
https://github.com/fgsfdsfgs/vitaXash3D 
PSVita port. Just for reference for anyone who have 3DS and some coding skills.
